components/board/orangepi/board.go

```diff
@@ -0,0 +1,27 @@
+// Package orangepi implements a orangepi based board.
+package orangepi
+
+import (
+	"errors"
+
+	"periph.io/x/host/v3"
+
+	"go.viam.com/rdk/components/board/genericlinux"
+	"go.viam.com/rdk/logging"
+)
+
+const modelName = "orangepi"
+
+func init() {
+	if _, err := host.Init(); err != nil {
+		logging.Global().Debugw("error initializing host", "error", err)
+	}
+
+	gpioMappings, err := genericlinux.GetGPIOBoardMappings(modelName, boardInfoMappings)
+	var noBoardErr genericlinux.NoBoardFoundError
+	if errors.As(err, &noBoardErr) {
+		logging.Global().Debugw("error getting orangepi GPIO board mapping", "error", err)
+	}
+
+	genericlinux.RegisterBoard(modelName, gpioMappings)
```

Comment:
These 3 paragraphs get copied and pasted around enough that we should probably make a helper function for it. but that can be done another time.